### ISSUE TITLE
bandit CI job failure on develop

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#4424]
## Description

Latest [bandit job failing on develop](https://circleci.com/gh/freedomofpress/securedrop/27356) (run in the `static-analysis-and-no-known-cves` CI job): 

```
#!/bin/bash -eo pipefail
make bandit
pip install --upgrade bandit && \
		bandit --recursive . --exclude admin/.tox,admin/.venv,admin/.eggs,molecule,testinfra,securedrop/tests,.tox,.venv -ll
Collecting bandit
  Downloading https://files.pythonhosted.org/packages/2b/27/1a5d835c4e09e182a0366d2007c232ba62f84406c75cce50860d20505cdc/bandit-1.6.0-py2.py3-none-any.whl (122kB)
    100% |████████████████████████████████| 122kB 2.9MB/s 
Exception:
Traceback (most recent call last):
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/basecommand.py", line 209, in main
    status = self.run(options, args)
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/commands/install.py", line 299, in run
    requirement_set.prepare_files(finder)
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/req/req_set.py", line 360, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/req/req_set.py", line 647, in _prepare_file
    set(req_to_install.extras) - set(dist.extras)
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2810, in extras
    return [dep for dep in self._dep_map if dep]
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2853, in _dep_map
    self.__dep_map = self._compute_dependencies()
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2886, in _compute_dependencies
    common = frozenset(reqs_for_extra(None))
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2883, in reqs_for_extra
    if req.marker_fn(override={'extra':extra}):
  File "/opt/circleci/.pyenv/versions/2.7.12/lib/python2.7/site-packages/pip/_vendor/_markerlib/markers.py", line 113, in marker_fn
    return eval(compiled_marker, environment)
  File "<environment marker>", line 1, in <module>
NameError: name 'platform_system' is not defined
You are using pip version 8.1.1, however version 19.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
make: *** [bandit] Error 2
Exited with code 2
```

**Comment by zenmonkeykstop** [comment]
This is probably an easy fix, but there's also a bug in bandit 1.60 ( https://github.com/PyCQA/bandit/issues/488 ) that will cause the job to fail due to unwanted checks. Will play with paths to see if there's a workaround

**Comment by shahpay** [comment]
Which docker image you are using, I had to replace pypi:bandit with only pytho:latest 
 and then install it
